posthog/constants.py

```diff
@@ -317,3 +317,6 @@ class FlagRequestType(StrEnum):
 PERMITTED_FORUM_DOMAINS = ["localhost", "posthog.com"]
 
 INVITE_DAYS_VALIDITY = 3  # number of days for which team invites are valid
+
+EU_INSTANCE_TEAM_ID = 1
+US_INSTANCE_TEAM_ID = 2
```

Comment:
Should these be constants or rather settings (thinking of self hosting folks)?

Reply:
Or better still, injected via env vars from charts